baseline/client/daily-tasks/daily-tasks.js

```diff
@@ -193,7 +193,7 @@ function taskForName(name, options) {
             const letter = availableLettersArr[rand];
             return new VerbalFluency(letter);
         case "verbal-learning":
-            return new VerbalLearning();
+            return new VerbalLearning(1, 1, () => 0);  // TODO: fix this!
```

Comment:
I didn't handle this verbal-learning update in daily-tasks correctly!

Reply:
I've fixed that in https://github.com/EmotionCognitionLab/pvs/pull/55/commits/9c9ab109d1664eb6072e5774013110d33253805e (and a few other commits that go along with it).